### PR TITLE
Remove Wildcard photos and update carousel

### PR DESCRIPTION
**Pull request by bryanlai00**

### CODE REVIEW

File: _data/carousel.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-- img: /img/carousel/group-2018.jpg
-  caption-header: StuyPulse with Wildcard on Stop Build Day.
+- img: /img/carousel/strat-2019.jpg
+  caption-header: StuyPulse's strategy team at the Central New York Regional!
+- img: /img/carousel/cny-2019.jpg
+  caption-header: StuyPulse at the Central New York Regional!
+- img: /img/carousel/tvr-2019.jpg
+  caption-header: StuyPulse at the New York Tech Valley Regional!
+- img: /img/carousel/Alfred.jpg
+  caption-header: This year's robot, Alfred!
```
Comment (justinkim) [inline _data/carousel.yml]: Can we keep this caption evergreen? e.g. `Our 2019 season robot, Alfred!`

File: about/history/2018/index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+flickr-collection-url: https://www.flickr.com/photos/stuypulse/collections/72157664823336618/
+youtube-links:
+        - name: Robot Reveal
+          url: https://www.youtube.com/watch?v=hVL89F4Zv-A
+---
+Wildcard, named in honor of our new coach, Joe Blay, competed in our craziest, and by far wildest season. The season started off well, making it past the semifinals in our first two regionals, playing on the #2 alliance in both, before pulling off a true blue in the New York City regional, getting one of our students nominated onto dean’s list, and not dropping a single match. At champs we performed extremely well, getting picked by the number 2 alliance, before losing in semifinals due to technical problems. It marked the first year we were invited to IRI, where we placed 15th. We finished the year going undefeated at the Qianjing Interational Robotics Invitational.
+
+Our best quality in 2018 was how much we improved over the season. After our first competition, we quickly swapped out our acquirer with a much more effective design, and by champs we had created over 10 working autons that would score points without getting in the way of other teams auton routines.
```
Comment (justinkim) [inline about/history/2018/index.md]: This blurb needs a proofreading for grammar, punctuation, and capitalization of some things. Work with marketing on this!

File: _scripts/check-stuypulse-capitalization.sh [hunk 3]
```diff
@@ -2,8 +2,8 @@
 # Checks for instances where "StuyPulse" is incorrectly capitalized as "Stuypulse".
 
 grep -rn --color \
-    --exclude-dir={".git","_site","_scripts","vendor"}
-    Stuypulse .
+     --exclude-dir={".git","_site","_scripts","vendor"} \
```
Comment (justinkim) [inline _scripts/check-stuypulse-capitalization.sh]: Did your editor add these spaces?

Comment (bryanlai00) [reply _scripts/check-stuypulse-capitalization.sh]: Yeah it was vim

**Pull request merged**